_docs/concepts/what-is-grafeas/overview.md

```diff
@@ -1,223 +1,108 @@
+aggregate information about your project's components.
 
-**Occurrences**: An occurrence can be thought of as an instantiation of a note and describes how the note was found in a specific cloud resource or project (e.g., location, specific remediation steps, etc.), or what the results of a specific note were (e.g., the container images that resulted from a build). For example, an occurrence might report that the heartbleed OpenSSL bug (a possible Note) was found in a specific package of a container image, and include information about how to remedy the heartbleed bug based on the customer’s package.
+Grafeas divides metadata information into [_notes_](#notes), which contain
+high-level information, and [_occurrences_](#occurrences), which contain
+information about how and when notes occur in a particular project. This
+division allows third party metadata providers to create and manage metadata on
+behalf of many customers. It also allows for fine-grained access control of
+different types of metadata.
 
-An occurrence name should take the format `/projects/<project_id>/occurrences/<occurrence_id>` where the project_id would typically be different from the project where the note is created and the occurrence_id would be unique per occurrence-project, and would often be random. 
+## Definition of terms
 
-Write access to occurrences should only be granted to users who have access to link a note to the occurrence. Any users can have read access to occurrences. 
+### Notes
+
+A _note_ describes a high-level piece of metadata. Notes are generally found via
+analysis, and are things that occur multiple times across different projects.
+For example, you could create a note about a particular vulnerability after
+analyzing a Linux package. You would also use a note to store information about
+the builder of a build process. Notes are often owned and created by providers
+who analyze users' packages.
+
+Note names should follow the format `/projects/<project_id>/notes/<note_id>`.
+The note id needs to be unique per note-project, and informative if possible.
+It's generally preferable to store notes and occurrences in separate projects,
+allowing for more fine-grained access control.
+
+Notes should be editable only by the note owner, and read-only for users who
+have access to occurrences referencing them.
+
+### Occurrences
+
+An _occurrence_ is essentially an instantiation of a note. Occurrences describe
+project-specific details of a given note. For example, an occurrence of a note
+about a vulnerability would describe the package that the vulnerability was
+found in, specific remediation steps, and so on. Alternatively, an occurrence of
+a note about build details would describe the container images that resulted
+from a build.
+
+Occurrence names should follow the format
+`/projects/<project_id>/occurrences/<occurrence_id>`. The occurrence id must be
+unique per occurrence-project and is often random. Typically, occurrences are
```

Comment:
what is occurrence-project?

Reply:
See my response to the question about note-project -- same idea.